wire-ios/Wire-iOS/Sources/AppRootRouter.swift

```diff
@@ -428,7 +434,7 @@ extension AppRootRouter {
         }
 
         let needToShowDialog = appStateCalculator.wasUnauthenticated && !isTeamMember
-
+        let e2eIdentityCertificateUpdate = await userSession.e2eIdentityUpdateCertificateUpdateStatus()
```

Comment:
I think most of the changes in this file should be reverted.
Please try to remove `async` from `userSession.e2eIdentityUpdateCertificateUpdateStatus()`.

Reply:
done